content/hardware/03.nano/boards/nano/tutorials/nano-getting-started/nano-getting-started.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Getting Started with Arduino Nano
+description: A step-by-step guide to install the board package needed for the Arduino Nano board.
+author: Hannes Siebeneicher
+tags: [Nano, Installation, IDE]
+---
+
+To use the [Arduino Nano](/hardware/nano/) board, you will need to install the Arduino AVR Board Package.
+
+To install it, you will need a version of the Arduino IDE, which you can download from the [Arduino Software page](https://www.arduino.cc/en/software). In this guide, we will use the latest version of the IDE 2.
+
+## Software & Hardware Needed
+
+- [Arduino Nano](https://store.arduino.cc/products/arduino-nano)
+- [Arduino IDE](/software/ide-v2)
+
+***You can also use the [Web Editor](https://create.arduino.cc/editor) which comes with all Arduino boards pre-installed.*** 
+
+## Download & Install IDE
+
+1. First, we need to download the Arduino IDE, which can be done from the [Arduino Software page](https://www.arduino.cc/en/software/).
+2. Install the Arduino IDE on your local machine.
+3. Open the Arduino IDE.
+
+![The Arduino IDE.](./assets/open-ide.png)
+
+## Install Board Package
+
+The [Arduino Core for AVR devices](https://github.com/arduino/ArduinoCore-avr) comes preinstalled with the IDE, so no additional installation is necessary to get started.  To use a different version than the latest, open the "Board Manager" from the left-hand menu. Search for Nano and install the version you want to use.
+
+![Arduino AVR Board Package](./assets/install-nano-core.png)
```

Comment:
I think we should change this screenshot to a close up, it's too small to see much of the text in this format. The changed screenshot should also show the drop down menu to select version being open imo.